tut/installing-appliance.md

```diff
@@ -29,44 +29,38 @@ The following instructions guide you through all relevant steps to install the A
 
 ### 1. Update your system
 
-Run the following commands to ensure that all installed packages on your system are up-to-date:
+On your machine, run the following commands to ensure that all installed packages on your system are up-to-date:
 
     sudo apt update
     sudo apt upgrade
 
-### 2. Install dependencies
-
-The Anbox Cloud Appliance requires LXD >= 5.0. Check which version you have currently installed:
-
-    lxd --version
-
-If LXD is not installed, run:
-
-    snap install --channel=5.0/stable lxd
-
-If LXD is already installed but the version is older than 5.0, run:
-
-    snap refresh --channel=5.0/stable lxd
-
 <a name="attach-ubuntu-pro"></a>
-### 3. Attach your machine to the Ubuntu Pro subscription
+### 2. Attach your machine to the Ubuntu Pro subscription
 
 The Anbox Cloud Appliance requires a valid Ubuntu Pro subscription.
 
 Before installing the appliance, you must attach the machine on which you want to run the Anbox Cloud Appliance to your Ubuntu Pro subscription. To do so, run the following command, replacing *<pro_token>* with your Ubuntu Pro token:
 
     sudo ua attach <pro_token>
 
-### 4. Install the snap
+<a name="enable-anbox-service"></a>
+### 3. Enable the `anbox-cloud` service using the Ubuntu Pro client.
+
+On your machine, run the following command to install the Anbox Cloud Appliance and additional dependencies.
+
+    pro enable anbox-cloud
 
-Run the following command to install the `anbox-cloud-appliance` snap, which handles the installation and deployment of the Anbox Cloud Appliance:
+Running this command does the following:
 
-    sudo snap install --classic anbox-cloud-appliance
+1. Install the following tools and dependencies, if they are not installed already:
+    * snapd
+    * Anbox Management Client (AMC)
+    * LXD
```

Comment:
I feel like this is a bit too short for a tutorial. Or should that be installed automatically from the pro enable command above?

Reply:
Running the `pro enable` command does all those automatically. I can see that using the imperative verb form can be confusing and hence I have changed it from "Install" to "Installs" which implies the action is being done by the command.